correios/client.py

```diff
@@ -109,9 +110,11 @@ def _call(self, method_name, *args, **kwargs):
         method = getattr(self.sigep, method_name)
         try:
             return method(*args, **kwargs)
+        except ConnectTimeout:
+            raise ConnectTimeoutError("Timeout connection error ({} seconds)".format(self.timeout))
         except Fault as exc:
-            if 'autenticacao' in str(exc):
-                raise AuthenticationError
+            if "autenticacao" in str(exc):
```

Comment:
Algum motivo das aspas duplas?

Reply:
Just to follow a pattern. I prefer single quotes but almost all the file use double quotes.